website/common/locales/en/generic.json

```diff
@@ -289,6 +289,16 @@
   "messages": "Messages",
   "emptyMessagesLine1": "You don't have any messages",
   "emptyMessagesLine2": "Send a message to start a conversation!",
+  "PMEnable": "Enable",
+  "PMEnableFull": "Enable private messages",
+  "PMDisable": "Disable",
+  "PMDisableFull": "Disable private messages",
+  "PMDisableConfirmation": "Are you sure you want to disable private messages? No one will able to send you messages, though you can enable private messages at any time.",
+  "PMEnabledOptPopoverText": "When private messages are disabled, no one can send them to you, though you can enable this option at any time.",
+  "PMDisabledOptPopoverText": "Enable private messages so you can chat again!",
+  "disabledPMCaptionTitle": "Private messages are disabled",
+  "disabledPMCaptionLine1": "Private messages are disabled and no one can send them to you.",
+  "disabledPMCaptionLine2": "You can enable this option at any time.",
```

Comment:
Can you move these strings to groups.json and place them just under the `optOutPopover` string, which is around line 135? That will keep all the texts about PMs together. You can then delete `optOutPopover` since it's no longer used.